spec/system/signup_spec.rb

```diff
@@ -16,6 +29,43 @@
 
       signup_modal.click_create_account
       expect(page).to have_css(".account-created")
+
+      mail = ActionMailer::Base.deliveries.first
+      expect(mail.to).to contain_exactly("johndoe@example.com")
+      activation_link = mail.body.to_s[%r{/u/activate-account/\S+}]
+
+      visit activation_link
+
+      find("#activate-account-button").click
+      find(".perform-activation .continue-button").click
+
+      expect(page).to have_current_path("/")
+      expect(page).to have_css(".header-dropdown-toggle.current-user")
```

Comment:
we could have or already have a page object for this

Reply:
That's the invitation modal, which is different from the normal signup modal. I didn't create a page object because I only needed it once. I can create one in the meantime, but I think we might want to use the same modal/page for both at some point

Reply:
ok